srcpkgs/firefox/files/xvfb-run.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/sh
```

Comment:
Shouldn't we have this file as part of xorg-server-xfvb

Reply:
Yes, it's just here for now to test if all of this even works.